package.json

```diff
@@ -21,7 +21,7 @@
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/salesforce/salesforcedx-vscode-mobile.git"
+        "url": "https://github.com/salesforce/salesforcedx-vscode-mobile"
```

Comment:
don't really need the .git in the url, and then can use it for the full md file link generation

Reply:
An alternative would be to use the [homepage](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#homepage) field of the configuration.

Reply:
Or maybe we just do it, since technically with GitHub repos you can clone the repo with `git clone` using the root URL (i.e. minus the `.git` extension), and it works. I guess as long as it works for your VCS provider, it's fine.

Reply:
The `homepage`  is `https://github.com/salesforce/salesforcedx-vscode-mobile/blob/main/README.md`. If we change it to `https://github.com/salesforce/salesforcedx-vscode-mobile/`, will that change the UI content for mobile extension?

<img width="1631" alt="image" src="https://github.com/user-attachments/assets/ae0c50a3-9150-4c2c-9a2a-be51d29cff5d" />

Reply:
Possibly. Yeah I'd say just leave it. As long as the URL can be used to clone the repo (which it can), I'd say that covers our bases.